# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../guppi/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 compile fixes.

  - Rene Rebe <rene@exactcode.de>

--- ./src/demo.c.orig	2004-05-13 20:57:44.936557856 +0200
+++ ./src/demo.c	2004-05-13 20:57:51.225601776 +0200
@@ -451,6 +451,7 @@
     compass_pos = GUPPI_NORTH;
     break;
   default:
+    ;
   }
 
   guppi_element_state_set (state,
--- ./src/libguppiplot/guppi-layout-constraint.c.orig	2004-05-13 20:52:41.636666448 +0200
+++ ./src/libguppiplot/guppi-layout-constraint.c	2004-05-13 20:52:51.744129880 +0200
@@ -165,6 +165,7 @@
 
   default:
     /* Fall through, do nothing. */
+    ;
   }
 
   ti = term_info_new (type, factor, geom);
--- ./src/libguppiplot/guppi-root-group-item.c.orig	2004-05-13 20:54:49.332253768 +0200
+++ ./src/libguppiplot/guppi-root-group-item.c	2004-05-13 20:55:03.409113760 +0200
@@ -548,6 +548,7 @@
     return motion_notify_event (root, &(ev->motion));
 
   default:
+    ;
   }
 
   return FALSE;
@@ -777,6 +778,7 @@
   case ROOT_GROUP_RESIZE_NONE:
   default:
     /* insure we do nothing */
+    ;
   }
 
   if (please_resize_canvas)
--- ./src/libguppiplot/guppi-text-block.c.orig	2004-05-13 20:51:20.957931480 +0200
+++ ./src/libguppiplot/guppi-text-block.c	2004-05-13 20:51:38.896204448 +0200
@@ -900,6 +900,7 @@
 
     default:
       /* Do nothing. */
+      ;
     }
 
     render_stack_evolve (stack, tt);
--- ./src/libguppiplot/guppi-view-interval.c.orig	2004-05-13 20:49:49.691806040 +0200
+++ ./src/libguppiplot/guppi-view-interval.c	2004-05-13 20:50:09.553786560 +0200
@@ -227,6 +227,7 @@
 #endif
 
   default:
+    ;
   }
 
   return TRUE;
--- ./libguppitank/guppi-object-barchart.c.orig	2004-05-13 21:07:35.963708072 +0200
+++ ./libguppitank/guppi-object-barchart.c	2004-05-13 21:07:48.188849568 +0200
@@ -701,6 +701,7 @@
     break;
 
   default:
+    ;
   }
 }
 
@@ -733,6 +734,7 @@
     break;
 
   default:
+    ;
   }
 }
 
